pythonnet.sln

```diff
@@ -34,6 +34,13 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 		setup.py = setup.py
 	EndProjectSection
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "conda.recipe", "conda.recipe", "{7FD2404D-0CE8-4645-8DFB-766470E2150E}"
```

Comment:
We don't maintain a conda recipe anymore.

Reply:
Removed. Looks like some VS glitch with solution file on branch change.